extensions-core/druid-kerberos/src/main/java/io/druid/security/kerberos/KerberosEscalator.java

```diff
@@ -57,6 +57,7 @@ public HttpClient createEscalatedClient(HttpClient baseClient)
   @Override
   public AuthenticationResult createEscalatedAuthenticationResult()
   {
-    return new AuthenticationResult(internalClientPrincipal, authorizerName, null);
+    return new AuthenticationResult(internalClientPrincipal, authorizerName, "kerberos", null);
```

Comment:
user may not have authenticator configured by name "kerberos", 
this should not be hardcoded
e.g for configs - 
druid.auth.authenticatorChain=["MyKerberosAuthenticator"]
druid.auth.authenticator.MyKerberosAuthenticator.type=kerberos

this should be set to MyKerberosAuthenticator I think.